dio/lib/src/adapters/io_adapter.dart

```diff
@@ -116,7 +116,11 @@ class IOHttpClientAdapter implements HttpClientAdapter {
       });
     } on SocketException catch (e) {
       if (!e.message.contains('timed out')) {
```

Comment:
Can you swap the condition here? That would make it a little bit easier to read.